### PR TITLE
Added pnpm workflows

### PR DESCRIPTION
**Pull request by timscodebase**
## Discord username (optional) please include so we can attribute you with our Contributor role (like so elvis#4747)
Tithos
## Description of changes (updated or new workflows)
I added workflows for pnpm

### CODE REVIEW

File: specs/pnpm/uninstall_a_local_pnpm_package.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Uninstall a local NPM package
```
Comment (alokedesai) [inline specs/pnpm/uninstall_a_local_pnpm_package.yaml:2]: nit: Could you include `pnpm` in the title for this workflow as well?

File: specs/pnpm/update_pnpm_to_the_latest_version.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+name: Update NPM to the latest version
```
Comment (alokedesai) [inline specs/pnpm/update_pnpm_to_the_latest_version.yaml:2]: same here about including `pnpm` in the title

File: specs/pnpm/update_pnpm_to_the_latest_version.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+command: pnpm update -g pnpm
+tags:
+  - pnpm
+description: Updates pnpm to the latest version .
```
Comment (alokedesai) [inline specs/pnpm/update_pnpm_to_the_latest_version.yaml:6]: epic nit: remove extra trailing space before the period

**Review by alokedesai — APPROVED** [review]
This looks great, thank you so much for contributing!

File: specs/pnpm/uninstall_a_local_pnpm_package.yaml (hunk 1 above)
Comment (timscodebase) [reply specs/pnpm/uninstall_a_local_pnpm_package.yaml:2]: Awesome thanks!

File: specs/pnpm/list_all_globally_installed_pnpm_packages.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+name: List all globally installed NPM packages
```
Comment (alokedesai) [inline specs/pnpm/list_all_globally_installed_pnpm_packages.yaml:2]: sorry missed this before, but I think this one also needs to be updated

**Review by alokedesai — APPROVED** [review]

**Comment by alokedesai** [comment]
🚢🚢🚢🚢

**Comment by WebRuin** [comment]
Is there something I can do to fix the failed test?

**Comment by alokedesai** [comment]
The tests are failing because these files were incorrectly moved. Mind reverting these changes? Once they are reverted, CI should start passing again
image

**Comment by alokedesai** [comment]
Hey @timscodebase mind addressing the last remaining comment! Would love to either get this merged or close out if it's no longer relevant

**Comment by elviskahoro** [comment]
@timscodebase I'm making a pass through our open PRs. Just wanted to bump this convo w.r.t Aloke's prior feedback

**Pull request merged**